colin-api/src/colin_api/models/filing.py

```diff
@@ -1268,6 +1283,42 @@ def is_filing_type_match(cls, filing: Filing, filing_type: str, filing_sub_type:
         """Return whether filing has specificed filing type and filing sub-type."""
         return filing.filing_type == filing_type and filing.filing_sub_type == filing_sub_type
 
+    @classmethod
+    def _process_continuation_out(cls, cursor, filing):
+        """Process continuation out."""
+        corp_num = filing.get_corp_num()
+
+        cont_out = ContOut()
+        cont_out.corp_num = corp_num
+        cont_out.start_event_id = filing.event_id
+        cont_out.cont_out_dt = filing.body.get('continuationOutDate')
+        cont_out.home_company_nme = filing.body.get('legalName')
+
+        foreign_jurisdiction = filing.body.get('foreignJurisdiction')
+        country_code = foreign_jurisdiction.get('country').upper()
+        region_code = (foreign_jurisdiction.get('region') or '').upper()
+        if country_code == 'CA':
+            if region_code == 'FEDERAL':
+                cont_out.can_jur_typ_cd = 'FD'
+            else:
+                cont_out.can_jur_typ_cd = region_code
+        else:
+            cont_out.can_jur_typ_cd = 'OT'
+            cont_out.othr_juri_desc = \
+                f'{country_code}, {region_code}' if region_code else country_code
+
+        ContOut.create_cont_out(cursor, cont_out)
+
+        continued_to = cont_out.othr_juri_desc if cont_out.can_jur_typ_cd == 'OT' else cont_out.can_jur_typ_cd
+        cont_out_dt_str = datetime.datetime.fromisoformat(cont_out.cont_out_dt).strftime('%B %-d, %Y')
+        cls._insert_ledger_text(
+            cursor,
+            filing,
+            f'CONTINUED TO {continued_to} EFFECTIVE {cont_out_dt_str} UNDER THE NAME "{cont_out.home_company_nme}"'
```

Comment:
Out of curiosity, is this displayed anywhere?

Reply:
not sure. might be in dashboard

Reply:
Screenshot?

Reply:
i dont have access